Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -182,6 +180,28 @@ - (void)formChanged:(NSNotification *)notification
         wantsWheelEventsChanged();
 }
 
+void UnifiedPDFPlugin::installAnnotationContainer()
+{
+    ASSERT(m_annotationContainer);
+
+    Ref document = m_element->document();
+
+    if (supportsForms()) {
+        RefPtr { document->bodyOrFrameset() }->appendChild(*m_annotationContainer);
```

Comment:
won’t this crash in release mode if `m_annotationContainer` is somehow empty?

Reply:
Correct.. right now it's called exclusively after annotation container setup, but I suppose we should be a bit more robust against the risk of crashing here? Any suggestions?

Reply:
I usually just do something like 

```
if (!x) {
  ASSERT_NOT_REACHED()
  return
}
```